README.md

```diff
@@ -406,6 +539,252 @@ minio:
+      - cleanup_bc: {} # A sample built-in command that triggers stop on all nodes and clean up.
+      - start: ['sharder-1']
+      - start: ['miner-1', 'miner-2', 'miner-3']
+      - wait_phase: 
+          phase: 'contribute'
+      - stop: ['miner-1']
+      - start: ['miner-1']
+      - wait_view_change:
+          timeout: '5m'
+          expect_magic_block:
+            miners: ['miner-1', 'miner-2', 'miner-3']
+            sharders: ['sharder-1']
+  - name: "Miner: 100 (switch to share)"
+    flow:
+    ...
+  - name: "All blobber tests"
+    flow:
+      - command:
+          name: 'build_test_blobbers' # Sample custom command that executes `build_test_blobbers`
+    ...
+...
+```
+
+## Getting Started
+
+### Prerequisites
+
+Docker and Git must be installed in order to run the tests .
+
+Install Git using the following command:
+
+```
+sudo apt install git
+```
+
+Docker installation instructions can be found [here](https://docs.docker.com/engine/install/).
+
+### Cloning the repository and Building Base Image
+
+Clone the 0chain repository:
+
+```
+git clone https://github.com/0chain/0chain.git
+```
+
+Build miner docker image for integration test
+
+```
+(cd 0chain && ./docker.local/bin/build.miners-integration-tests.sh)
+```
+
+Build sharder docker image for integration test
+
+```
+(cd 0chain && ./docker.local/bin/build.sharders-integration-tests.sh)
+```
+
+NOTE: The miner and sharder images are designed for integration tests only. If wanted to run chain normally, rebuild the original images.
+
+```
+(cd 0chain && ./docker.local/bin/build.sharders.sh && ./docker.local/bin/build.miners.sh)
+```
+
+Confirm that view change rounds are set to 50 on `0chain/docker.local/config.yaml`
+
+```
+    start_rounds: 50
+    contribute_rounds: 50
+    share_rounds: 50
+    publish_rounds: 50
+    wait_rounds: 50
+```
+
+### Running standard tests
+
+Run miners test
+
+```
+(cd 0chain && ./docker.local/bin/start.conductor.sh miners)
+```
+
+Run sharders test
+
+```
+(cd 0chain && ./docker.local/bin/start.conductor.sh sharders)
+```
+
+### Running view-change tests
+
+1. Set `view_change: true` on `0chain/docker.local/config.yaml`
+2. Run view-change tests
+
+```
+(cd 0chain && ./docker.local/bin/start.conductor.sh view-change-1)
+(cd 0chain && ./docker.local/bin/start.conductor.sh view-change-2)
+(cd 0chain && ./docker.local/bin/start.conductor.sh view-change-3)
+```
+
+### Running blobber tests
+
+Blobber tests require cloning of below services.
+
+ [blobber](https://github.com/0chain/blobber)
+
+```
+git clone https://github.com/0chain/blobber.git
+```
+
+[zboxcli](https://github.com/0chain/zboxcli)
+
+```
+git clone https://github.com/0chain/zboxcli.git
+```
+
+ [zwalletcli](https://github.com/0chain/zwalletcli)
+
+```
+git clone https://github.com/0chain/zwalletcli.git
+```
+
+ [0dns](https://github.com/0chain/0dns)
+
+```
+git clone https://github.com/0chain/0dns.git
+```
+
+Confirm the directories
```

Comment:
Is this necessary? We are allowed to assume the user will notice if one of his `git clones` error out. Plus the user will notice later if there is an issue as one of the `cd` commands will fail.

Could also be worded better.